src/hooks/useKonamiCheatCode.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useCallback, useState } from "react";
+import useKonamiCheatCode from "./useKonamiCheatCode";
+
+export default {
+  title: "Hooks/useKonamiCheatCode",
+  argTypes: {},
+};
+
+export const Default: React.FC = () => {
+  const [correctCodeEntered, setCorrectCodeEntered] = useState(false);
+  const correctCodeHandler = useCallback(() => setCorrectCodeEntered(true), [setCorrectCodeEntered]);
+  useKonamiCheatCode(correctCodeHandler);
```

Comment:
Instead of requiring the consumer to memoize the handler lets put it inside the hook itself using `useRef`.